docs/source/gateway/index.rst

```diff
@@ -30,15 +30,15 @@ By storing API keys in one secure location, organizations can significantly enha
 security posture by minimizing the exposure of sensitive API keys throughout the system. It also
 helps to prevent exposing these keys within code or requiring end-users to manage keys safely.
 
-The gateway is designed to be flexible and adaptable, capable of easily defining and managing routes
-using a straightforward REST API. This enables the easy incorporation
+The gateway is designed to be flexible and adaptable, capable of easily defining and managing routes and
+rate limits using a straightforward REST API. This enables the easy incorporation
 of new LLM providers or provider LLM types into the system without necessitating changes to
 applications that interface with the gateway. This level of adaptability makes the Databricks AI Gateway
 Service an invaluable tool in environments that require agility and quick response to changes.
 
 This simplification and centralization of language model interactions, coupled with the added
-layer of security for API key management, make the Databricks AI Gateway service an ideal choice for
-organizations that use LLMs on a regular basis.
+layer of security for API key management and rate limiting, make the Databricks AI Gateway service an
+ideal choice for organizations that use LLMs on a regular basis.
```

Comment:
I would say "cost controls" more generally than "rate limiting"

Reply:
Changed to "cost controls"!